go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/openmarketplaceengine/openmarketplaceengine
 go 1.17
 
 require (
+	github.com/cristalhq/aconfig v0.16.8
```

Comment:
i guess we should remove github.com/spf13/viper along with previous config implementation.